digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -4,21 +4,25 @@ import fi.liikennevirasto.digiroad2.linearasset.PolyLine
 
 object GeometryUtils {
 
-  // Default value of minimum distance where locations are considered to be same
+  /** Default value of minimum distance where locations are considered to be same */
   final private val DefaultEpsilon = 0.01
+  /** If links are more than <i>MaxDistanceForConnectedLinks</i> apart, they are considered to being disconnected. */
   val MaxDistanceForConnectedLinks = 0.1
 
+  /** @return the first, and last the point of the given <i>geometry</i> as a tuple. */
   def geometryEndpoints(geometry: Seq[Point]): (Point, Point) = {
     val firstPoint: Point = geometry.head
     val lastPoint: Point = geometry.last
     (firstPoint, lastPoint)
   }
 
+  /** @return the first, and the last point of the given <i>geometry</i> as a sequence. */
   def geometrySeqEndPoints(geometry: Seq[Point]): Seq[Point] = {
     Seq(geometry.head, geometry.last)
   }
 
-  private def liesInBetween(measure: Double, interval: (Double, Double)): Boolean = {
+  /** @return True/False, whether the <i>measure</i> lays within the <i>interval</i>. */
+  private def laysInBetween(measure: Double, interval: (Double, Double)): Boolean = {
```

Comment:
Lies on oikeampi termi tälle

"lay is transitive and requires an object to act upon, and lie is intransitive, describing something moving on its own or already in position."

Reply:
Ha, opin juuri lisää englantia. (Ja we'll see, miten pitkäaikaiseen muistiin tallentuu nyt...) 
En tiennyt että lying on jotain muutakin kuin valehtelua :D
Korjataan takas! :)

Reply:
Korjattu